ts-magic-eight-ball/src/App.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState,ChangeEvent } from "react";
+import questionAnswer from './components/questionAnswer'
+import {Title,Button,Body,H2,Input,P} from "./style"
+
+const API_ENDPOINT = 'https://yesno.wtf/api';
+
+function App() {
+  const [question, setQuestion] = useState<string>("");
+  const [item, setItem] = useState<any>("");
```

Comment:
add api response's type by yourself and use it like `useState<AnswerResponse>()`

```ts
type AnswerResponse {
  // its properties
}
```